src/components/ButtonAddTransactions/ButtonAddTransactions.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import css from './ButtonAddTransactions.module.css';
+export const ButtonAddTransactions = ({ handleClick }) => {
```

Comment:
Na wstępie (wiem, że dokumentacja do projektu jest, jaka jest), ale:
Klikając w ButtonAddTransactions spodziewałbym się (patrząc na nazewnictwo przycisku), że będzie wykonywana akcja dodawania więcej niż jednej transakcji. Jednak wywołany jest modal dla dodawania jednej transakcji? 
Co sądzisz?
Ja bym tu zrobił ButtonAddTransaction.

Reply:
nie mogę sie nie zgodzić ale w dokumentacji mamy TButtonAddransactions które wywołuje komponent ModalAddTransaction , czyli "przycisk dodający transakcje" przenosi do Modala który dodaje transakcję . Jak uznasz, mogę zmienić tylko ze to nazwy komponentow i funkcji (co nie jest dużym problemem)

Reply:
zrobione, zmieniona nazwa ButtonAddTransaction